### PR TITLE
vscode: show release tag with along with the commit hash for RA version command

### PR DESCRIPTION
**Pull request by Veetaha**

### CODE REVIEW

**Comment by Veetaha** [comment]
Don't what's going on, @matklad, you better investigate)

**Comment by matklad** [comment]
Is this rebased?

**Comment by Veetaha** [comment]
It is

File: editors/code/src/commands/server_version.ts [hunk 1]
```diff
@@ -4,7 +4,12 @@ import { Ctx, Cmd } from '../ctx';
 
 export function serverVersion(ctx: Ctx): Cmd {
     return async () => {
-        const version = spawnSync(ctx.serverPath, ["--version"], { encoding: "utf8" }).stdout;
-        vscode.window.showInformationMessage('rust-analyzer version : ' + version);
+        const { stdout } = spawnSync(ctx.serverPath, ["--version"], { encoding: "utf8" });
+        const commitHash = stdout.slice(`rust-analyzer `.length).trim();
+        const { releaseTag } = ctx.config;
```
Comment (lnicola) [inline editors/code/src/commands/server_version.ts]:
```suggestion
        const { releaseTag } = ctx.config.package;
```

**Comment by Veetaha** [comment]
Thanks lnicola, good catch. This is very strange that `npm run package` doesn't display any error messages from `tsc`...

**Comment by lnicola** [comment]
Try `npm run vscode:prepublish`. I find it stranger that the error message doesn't end up in the CI logs.

**Pull request merged**